3rdparty/python/requirements.txt

```diff
@@ -27,7 +27,7 @@ pywatchman==1.4.1
 requests[security]>=2.5.0,<2.19
 scandir==1.2
 setproctitle==1.1.10
-setuptools==30.0.0
+setuptools==33.1.1
```

Comment:
You should bump lockstep over here:
https://github.com/Eric-Arellano/pants/blob/9a8ccd4916a11fdc6abe5941ff71f855d4ec003e/src/python/pants/backend/python/subsystems/python_setup.py#L36-L37

Reply:
Good catch. Fixed with latest commit. Thanks!